src/devices/cwd.ts

```diff
@@ -0,0 +1,12 @@
+const m = require('zigbee-herdsman-converters/lib/modernExtend');
```

Comment:
Please add to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts